package.json

```diff
@@ -31,7 +31,8 @@
     "conventional-changelog": "5.1.0",
     "conventional-changelog-cli": "4.1.0",
     "husky": "8.0.3",
-    "prettier": "3.0.3",
+    "license-checker-commit": "25.0.2",
+    "prettier": "2.8.8",
```

Comment:
prettier changed

Reply:
`pretty-quick` is incompatible with `prettier` v3 https://github.com/prettier/pretty-quick/issues/164